pyglotaran_extras/inspect/a_matrix.py

```diff
@@ -0,0 +1,165 @@
+    decimal_places: int = 3,
+) -> str:
+    """Create html multi header table from a-matrix.
+
+    Parameters
+    ----------
+    a_matrix: xr.DataArray
+        DataArray containing the a-matrix values and coordinates.
+    megacomplex_suffix: str
+        Megacomplex suffix used for the a-matrix data variable and coordinate names.
+    normalize_initial_concentration: bool
+        Whether or not to normalize the initial concentration. Defaults to False
+    decimal_places: int
+        Decimal places to display. Defaults to 3
+
+    Returns
+    -------
+    str
+        Multi header htm table representing the a-matrix.
+    """
+    species = a_matrix.coords[f"species_{megacomplex_suffix}"].values
+    # Crete a copy so normalization does not mutate the original values
+    initial_concentration = np.array(
+        a_matrix.coords[f"initial_concentration_{megacomplex_suffix}"].values
+    )
+    lifetime = a_matrix.coords[f"lifetime_{megacomplex_suffix}"].values
+
+    if normalize_initial_concentration is True:
+        initial_concentration /= initial_concentration.sum()
+
+    header = (
+        ["species<br>initial concentration<br>lifetime↓"]
+        + [
+            f"{sp}<br>{pretty_format_numerical(ic,decimal_places)}<br>&nbsp;"
+            for sp, ic in zip(species, initial_concentration)
+        ]
+        + ["Sum"]
+    )
+
+    data = [
+        pretty_format_numerical_iterable(
+            (lifetime, *amps, amps.sum()), decimal_places=decimal_places
+        )
+        for lifetime, amps in zip(lifetime, a_matrix.values)
+    ]
+    data.append(
+        pretty_format_numerical_iterable(
+            ("Sum", *a_matrix.values.sum(axis=0), a_matrix.values.sum()),
+            decimal_places=decimal_places,
+        )
+    )
+
+    return (
+        tabulate(
+            data, headers=header, showindex=False, tablefmt="unsafehtml", disable_numparse=True
+        )
+        .replace(" 0 ", "   ")
+        .replace(" 0<", "  <")
+        .replace(">0 ", ">  ")
+    )
+
+
+def show_a_matrixes(
+    result: ResultLike,
+    *,
+    normalize_initial_concentration: bool = False,
+    decimal_places: int = 3,
+    a_matrix_min_size: int | None = None,
+    expanded_datasets: tuple[str, ...] = (),
+    heading_offset: int = 2,
+) -> MarkdownStr:
+    """Show all a-matrixes of a result grouped by dataset and megacomplex name.
+
+    Each dataset is wrapped in a HTML details tag which is by default collapsed.
+
+    Parameters
+    ----------
+    result: ResultLike
+        Result or result dataset.
+    normalize_initial_concentration: bool
+        Whether or not to normalize the initial concentration. Defaults to False.
+    decimal_places: int
+        Decimal places to display. Defaults to 3.
+    a_matrix_min_size: int
+        Defaults to None.
+    expanded_datasets: tuple[str, ...]
+        Names of dataset to expand the details view for. Defaults to () which means no dataset
```

Comment:
```suggestion
        Names of dataset to expand the details view for. Defaults to empty tuple () which means no dataset
```

defaults to () could look like a type, perhaps add "empty tuple"